src/spiceypy/spiceypy.py

```diff
@@ -5185,6 +5185,29 @@ def ev2lin(et: float, geophs: Sequence[float], elems: Sequence[float]) -> ndarra
     return stypes.c_vector_to_python(state)
 
 
+def evsgp4(et: float, geophs: Sequence[float], elems: Sequence[float]) -> ndarray:
+    """
+    Evaluate NORAD two-line element data for earth orbiting
+    spacecraft. This evaluator uses algorithms as described
+    in Vallado 2006
+
+    https://naif.jpl.nasa.gov/pub/naif/toolkit_docs/C/cspice/evsgp4_c.html
+
+    :param et: Epoch in seconds past ephemeris epoch J2000.
+    :param geophs: Geophysical constants
+    :param elems: Two-line element data
+    :return: Evaluated state
+    """
+    et = ctypes.c_double(et)
+    assert len(geophs) == 8
+    geophs = stypes.to_double_vector(geophs)
+    assert len(elems) == 10
+    elems = stypes.to_double_vector(elems)
+    state = stypes.empty_double_vector(6)
+    libspice.ev2lin_(ctypes.byref(et), geophs, elems, state)
```

Comment:
shouldn't this call be to `libspice.evsgp4_c` instead?

Reply:
good catch